meson.build

```diff
@@ -93,9 +93,10 @@ endif
 
 prefixdir = get_option('prefix')
 bindir = join_paths(prefixdir, get_option('bindir'))
+libexecdir = join_paths(prefixdir, get_option('libexecdir'))
```

Comment:
as I said, not going to work. see https://github.com/systemd/casync/pull/37

Reply:
The explanation there is this:

> /usr/lib is really where shared libraries, as well as binaries of the primary arch belong. And /usr/lib64 is where the secondary arch shared libs belong.

Not really. At least on debian-based systems, it is preferred to install shared libraries under `/usr/lib/$triplet`, where `$triplet` is a gnu triplet (eg, `x86_64-linux-gnu`). This allows co-installing libraries from multiple architectures. But helper programs normally do not need to exist for any architecture but the primary, and thus do not benefit from this scheme. In fact, it can introduce problems, if it can ever happen that a program configured for `x86_64-linux-gnu` runs with the helper binaries configured for `i386-linux-gnu`, then it won't be found. Therefore, programs may be installled directly under `/usr/lib/$package`.

This can be seen in eg, systemd, where helper binaries live in `/lib/systemd`, but the shared library lives in `/lib/x86_64-linux-gnu/libsystemd.so.0`.

Casync does not currently ship any libraries, but according to [the initial announcement](http://0pointer.net/blog/casync-a-tool-for-distributing-file-system-images.html) it is planned to ship one, so I think it makes sense to decouple helper programs from the shared libraries.